src/Telegram.Bot/Polling/DefaultUpdateReceiver.cs

```diff
@@ -8,41 +8,32 @@ namespace Telegram.Bot.Polling;
 /// <summary>
 /// A simple <see cref="IUpdateReceiver"/>> implementation that requests new updates and handles them sequentially
 /// </summary>
+/// <remarks>
+/// Constructs a new <see cref="DefaultUpdateReceiver"/> with the specified <see cref="ITelegramBotClient"/>
+/// instance and optional <see cref="ReceiverOptions"/>
+/// </remarks>
+/// <param name="botClient">The <see cref="ITelegramBotClient"/> used for making GetUpdates calls</param>
+/// <param name="receiverOptions">Options used to configure getUpdates requests</param>
 [PublicAPI]
-public class DefaultUpdateReceiver : IUpdateReceiver
+public class DefaultUpdateReceiver(
+    ITelegramBotClient botClient,
+    ReceiverOptions? receiverOptions = default) : IUpdateReceiver
 {
     static readonly Update[] EmptyUpdates = [];
 
-    readonly ITelegramBotClient _botClient;
-    readonly ReceiverOptions? _receiverOptions;
-
-    /// <summary>
-    /// Constructs a new <see cref="DefaultUpdateReceiver"/> with the specified <see cref="ITelegramBotClient"/>
-    /// instance and optional <see cref="ReceiverOptions"/>
-    /// </summary>
-    /// <param name="botClient">The <see cref="ITelegramBotClient"/> used for making GetUpdates calls</param>
-    /// <param name="receiverOptions">Options used to configure getUpdates requests</param>
-    public DefaultUpdateReceiver(
-        ITelegramBotClient botClient,
-        ReceiverOptions? receiverOptions = default)
-    {
-        _botClient = botClient ?? throw new ArgumentNullException(nameof(botClient));
-        _receiverOptions = receiverOptions;
-    }
+    readonly ITelegramBotClient _botClient = botClient ?? throw new ArgumentNullException(nameof(botClient));
```

Comment:
You use `ReceiverOptions` without a separate private readonly field. Why not do the same with the bot client?